src/main/java/uk/gov/hmcts/divorce/divorcecase/model/GeneralApplicationType.java

```diff
@@ -15,14 +15,26 @@ public enum GeneralApplicationType implements HasLabel {
     @JsonProperty("deemedService")
     DEEMED_SERVICE("Deemed service"),
 
-    @JsonProperty("issueDivorceWithoutMarriageCertificate")
-    ISSUE_DIVORCE_WITHOUT_CERT("Issue divorce without marriage certificate"),
+    @JsonProperty("otherAlternativeServiceMethod")
+    OTHER_ALTERNATIVE_SERVICE_METHODS("Alternative service"),
 
     @JsonProperty("expedite")
     EXPEDITE("Expedite"),
 
-    @JsonProperty("otherAlternativeServiceMethod")
-    OTHER_ALTERNATIVE_SERVICE_METHODS("Other alternative service methods"),
+    @JsonProperty("issueDivorceWithoutMarriageCertificate")
+    ISSUE_DIVORCE_WITHOUT_CERT("Issue divorce without marriage certificate"),
+
+    @JsonProperty("orderOnFilingOfAnswers")
+    ORDER_ON_FILLING_OF_ANSWERS("Order on filing of Answers"),
+
+    @JsonProperty("permissionOnDaOot")
+    PERMISSION_ON_DA_OOT("Permission on FO OOT"),
+
+    @JsonProperty("disclosureViaDwp")
+    DISCLOSURE_VIA_DWP("Disclosure via DWP"),
+
+    @JsonProperty("amendApplication")
+    AMEND_APPLICATION("Amend Application"),
```

Comment:
Do we also need to add `Order application without m/c` as a type? I might have missed it

Reply:
This is similar to Issue divorce without M/C. So consolidated to one option and clarified the final list with Izzy on the jira ticket.